Desktop/widgets/listmodel.h

```diff
@@ -131,8 +137,7 @@ public slots:
 
 			JASPListControl*				_listView = nullptr;
 			Terms							_terms;
-			bool							_areTermsVariables		= true;
-			bool							_areTermsInteractions	= false;
+			QStringList						_columnsUsedForLabels;
```

Comment:
It seems to me you only use this to look up things, I would suggest you turn it into a `set` then. 

You could add the inverse of https://github.com/jasp-stats/jasp-desktop/blob/cad06e46416bec1de0183cdcdeba06877ef844d8/Desktop/utilities/qutils.h#L43 there for it